recipes/aehmc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: aehmc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/aehmc/aehmc-{{ version }}.tar.gz
+  sha256: 1e3e648198baf4c610c09f77d91a3affbdb83cb1f212a6c327885cf4300b2ae2
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - python >=3.6
+    - numpy >=1.18.1
+    - scipy >=1.4.0
+    - aesara >=2.1.1
+    - aeppl >=0.0.5
+
+test:
+  imports:
+    - aehmc
```

Comment:
It looks like `import aehmc` doesn't actually do anything meaningful. Can you add a few more import tests to validate that the package isn't broken?

Reply:
Good point; @rlouf, which modules should we import?

Reply:
`aehmc.hmc` and `aehmc.nuts` would makje sense.